tests/solver_integration/cpp_solver/hello.cpp

```diff
@@ -0,0 +1,26 @@
+// CoSimulation includes
+#include "co_sim_io.hpp"
+
+// move this macro to a checks.hpp in tests folder?
```

Comment:
I would leave those examples really self-contained, hence I would copy this to every example